Crop Production tests broken on main

Two tests are failing on main, presumably due to a dependency update.
```
FAILED tests/test_crop_production.py::CropProductionTests::test_crop_production_percentile - AssertionError: DataFrame.iloc[:, 33] (column name="ca_yield_50th") are different

DataFrame.iloc[:, 33] (column name="ca_yield_50th") values are different (100.0 %)
[index]: [0, 1, 2]
[left]:  [2270468796, 997755421, 5264062838]
[right]: [2270468796.3867188, 997755421.0205078, 5264062837.890625]
FAILED tests/test_crop_production.py::CropProductionTests::test_crop_production_regression - AssertionError: DataFrame.iloc[:, 17] (column name="mg_modeled") are different

DataFrame.iloc[:, 17] (column name="mg_modeled") values are different (100.0 %)
[index]: [0, 1, 2]
[left]:  [4177775457, 1956114896, 9988580561]
[right]: [4177775456.665039, 1956114895.965576, 9988580596.191406]
```

--- a/tests/test_crop_production.py
+++ b/tests/test_crop_production.py
@@ -61,7 +61,8 @@ class CropProductionTests(unittest.TestCase):
         agg_result_table = pandas.read_csv(
             agg_result_table_path)
         pandas.testing.assert_frame_equal(
-            expected_agg_result_table, agg_result_table, check_dtype=False)
+            expected_agg_result_table, agg_result_table,
+            check_dtype=False, check_exact=False)
 
         expected_result_table = pandas.read_csv(
             os.path.join(TEST_DATA_PATH, 'expected_result_table.csv')
@@ -316,7 +317,8 @@ class CropProductionTests(unittest.TestCase):
         agg_result_table = pandas.read_csv(
             os.path.join(args['workspace_dir'], 'aggregate_results.csv'))
         pandas.testing.assert_frame_equal(
-            expected_agg_result_table, agg_result_table, check_dtype=False)
+            expected_agg_result_table, agg_result_table,
+            check_dtype=False, check_exact=False)
 
         result_table_path = os.path.join(
             args['workspace_dir'], 'result_table.csv')

